docs/how-to-setup-home-server.md

```diff
@@ -0,0 +1,82 @@
+Installation for vector on a home server
+========================================
+
+Install or update `node.js` so that your `npm` is at least at version `2.0.0`
+
+Follow those steps:
+
+```
+cd ~/matrix
+git clone https://github.com/vector-im/vector-web.git
+cd vector-web
+git checkout develop
```

Comment:
We shouldn't be telling people to check out develop as a matter of course: this defeats the point of having releases. Happy for there to be instructions for running the develop branch in a separate place.